rcsb/db/wf/RepoLoadWorkflow.py

```diff
@@ -277,6 +280,56 @@ def removeAndRecreateDbCollections(self, op, **kwargs):
 
         return ok
 
+    def getPdbImgsFormattedList(self, hD, **kwargs):
+        pdbIdsTimestamps = {}
+        for idVal in hD:
+            datetimeObject = datetime.datetime.strptime(hD[idVal], "%Y-%m-%dT%H:%M:%S%z")
+            pdbIdsTimestamps[idVal.lower()] = datetimeObject
+        idL = []
+        if kwargs.get("updateAllImages"):
+            for idVal in pdbIdsTimestamps:
+                path = idVal + ".bcif" if kwargs.get("noBcifSubdirs") else idVal[1:3] + "/" + idVal + ".bcif"
+                idL.append(f"{idVal} {path} experimental")
+        else:
+            for idVal, timestamp in pdbIdsTimestamps.items():
+                path = idVal + ".bcif" if kwargs.get("noBcifSubdirs") else idVal[1:3] + "/" + idVal + ".bcif"  # idVal[1:3] + "/" + idVal + ".bcif"
+                bcifFile = os.path.join(kwargs.get("BcifBaseDir"), path)
+                if Path.exists(bcifFile):
+                    t1 = Path.stat(bcifFile).stMtime
+                    t2 = timestamp.timestamp()
+                    if t1 < t2:
+                        idL.append(f"{idVal} {path} experimental")
```

Comment:
I'm not sure how this part is working for you...I think you need to create a Path file object to apply `Path.stat()` on. Also, I think you mean `.st_mtime`, not `stMtime`. 

I.e., 
```python
from pathlib import Path
bcifFilePath = Path(bcifFile)
t1 = Path.stat(bcifFilePath).st_mtime
```